lib/server.ml

```diff
@@ -69,55 +69,57 @@ let connect channel ?offer () =
         readfn payload
         >>= fun () -> return (hdr.OptionRequestHeader.ty, payload)
   in
```

Comment:
Just in case you don't know: when viewing this commit's diff on github, adding `?w=1` to the URL may be helpful (hides whitespace changes).